alacritty_terminal/src/display.rs

```diff
@@ -226,7 +227,14 @@ impl Display {
 
         // We should call `clear` when window is offscreen, so when `window.show()` happens it
         // would be with background color instead of uninitialized surface.
-        window.swap_buffers()?;
+        // We also use this opportunity to see if buffer damage is supported.
+        let damage_supported = match window.swap_buffers_with_damage(&[]) {
+            Ok(_) => true,
```

Comment:
Yes. It might not seem immediately obvious—it should be documented here or in glutin. I will never be a fan of gl interfaces.

The glutin part is still WIP (https://github.com/rust-windowing/glutin/pull/1207).